docs/rfcs/0004-suggested-excluded-apps-tech-spec.md

```diff
@@ -0,0 +1,153 @@
+    -   This will be run by the app for desktop, and in the daemon on Android. (iOS does not have split tunnel.)
+        -   After running the job, we will create timer for noon the following day to run the job next. Additionally, we'll record the time for this job as a setting, and we'll check this timestamp on app/daemon launch - if the deadline has passed we'll immediately run the task.
+        -   We're not concerned if the user's clock shifts. The vast majority of these daily checks will not result in any notification to the user (except in the rare case that the user consistently adds new apps daily that happen to be on the suggested list). The worst case scenario would be that a user gets two notifications in a 24 hour period. This isn't the planned user interaction, but given the low likelihood (neither clock changes nor the installation of new suggested apps is a common event) it's not worth building a system that considers this.
+-   (3-5 points) Is there one or more new apps?
+    -   Check against setting - if we have ever sent a notification for this app before, don't send one (this should be done off the app's domain name or something, not marketing name - that can change)
+    -   Pop an OS-level notification (on Android, use a suggestion channel). The text is intentionally generic, and makes sense if there is one new app or multiple.
+    -   Add new app(s) to "have sent notification" list
+-   (1 point) Add this notification type to the settings screen that allows per-type opt out (App permissions -> Notifications)
+-   (2 points) Ensure tapping notification leads to correct screen
+-   (2 points) Add 3 dev menu options:
+    - Pull a new suggested apps list immediately
+    - Clear the "have sent notification" list.
+    - Immediately run the daily check job.
+
+### Improve existing split tunnel screen (10-16 points total)
+[Figma mocks](https://www.figma.com/file/UZYzma7hlcfE5ke3z8jGbN/App-exclusions-suggestions?type=design&node-id=196-6366&mode=design&t=RL1hdfBQLMS1rKVa-0)
+
+<img src="./images/0004-03.png" width=500 alt="Updated flow for existing App Exclusion screen">
+
+-   (2 points) Run the task on the launch of screen
+    -   Update the "have seen these apps" settings list with all suggested apps. (If a new suggested app is install and the Settings screen is opened by a user before the daily job runs to send a notification, we thus never send a notification. This behavior was confirmed by Santiago.)
+-   (1-2 points) Add recommended section (only if 1 more more apps on the system are suggested). Since we've just updated the settings list of recommended apps on the system (in prior bullet point), use this settings list - if any system app we discover also appears on that list, it goes in the recommended section.
```

Comment:
```suggestion
-   (1-2 points) Add recommended section (only if 1 or more apps on the system are suggested). Since we've just updated the settings list of recommended apps on the system (in prior bullet point), use this settings list - if any system app we discover also appears on that list, it goes in the recommended section.
```